_data/simple-icons.json

```diff
@@ -4129,6 +4129,11 @@
             "hex": "FF3008",
             "source": "https://www.doordash.com/about/"
         },
+        {
+            "title": "Dota 2",
```

Comment:
Do you know if the first version of the game used the same logo? If so, we could remove the number to keep the title as "Dota" only. We could also add an "aka" attribute with "Defense of the Ancients".

Reply:
This logo is only used for Valve's Dota 2. The Warcraft3 mod used a different one.

The full title "Defense of the Ancients" is usually associated with the first version, although Valve also owns the trademark. ([source](https://liquipedia.net/dota2/Defense_of_the_Ancients)).